lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
+                    {'url': this.options.accountSettingsPageUrl}
+                );
                 if (this.profileIsPrivate) {
-                    this._super(interpolate_text(
-                        gettext("You must specify your birth year before you can share your full profile. To specify your birth year, go to the {account_settings_page_link}"),
-                        {'account_settings_page_link': accountSettingsLink}
-                    ));
+                    this._super(
+                        HtmlUtils.interpolateHtml(
+                            gettext("You must specify your birth year before you can share your full profile. " +
```

Comment:
Unfortunately you can't break ``gettext`` calls across multiple lines like this, because the scraper doesn't understand it. Instead you need to do the following:

    gettext('You must specify your birth year before you can share your full profile. To specify your birth year, go to the {account_settings_page_link}'),  // jshint ignore:line